packages/template-retail-react-app/app/pages/account/profile.jsx

```diff
@@ -321,7 +310,7 @@ const PasswordCard = () => {
                             <FormActionButtons
                                 onCancel={() => {
                                     setIsEditing(false)
-                                    passwordHeading?.focus()
+                                    headingRef?.current?.focus()
```

Comment:
I smoked test the change on your PR, the a11y is broken on profile page. E.g
After I cancel the password form, it is supposed to auto focus on the Password title, but it is not the case for your branch and the form is not reset. When I opened the form again, it still contains old input texts

Reply:
Ok.. I fixed both of those issues.